duckduckgo/Android
# Translations for Site Permission when Ask is disabled

Task/Issue URL: https://app.asana.com/0/488551667048375/1203428655709483/f

### Description
Translations for `sitePermissionsRemoveAllWebsitesSnackbarText` and `permissionsPerWebsiteAskDisabledSetting` keys

### Steps to test this PR

- Install from this branch
- Change device language
- Go to Settings > Site Permissions
- Toggle off any of the permissions
- Tap on a website you granted permissions before
- [x] Check "Ask" disabled for all sites is translated

### No UI changes
